api/swagger.yaml

```diff
@@ -1106,6 +1106,8 @@ definitions:
         type: "boolean"
       Attachable:
         type: "boolean"
+      Ingress:
+        type: "boolean"
```

Comment:
Can you add a mention of these changes to the API version-history? https://github.com/docker/docker/blob/master/docs/api/version-history.md#v128-api-changes